locales/locales.json

```diff
@@ -2059,7 +2057,46 @@
   },
   "search": {
     "de": {
-      "searchPlaceholder": "Suche"
+      "searchCategory": "Suche nach einer Kategorie"
```

Comment:
As we call categories now `Local Information`, this is probably wrong. Not sure and also not part of your task, so feel free to ignore this.